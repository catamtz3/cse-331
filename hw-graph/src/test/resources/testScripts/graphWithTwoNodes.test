# Create a graph
CreateGraph onlyTwo

#Add two nodes to the graph
AddNode onlyTwo a
AddNode onlyTwo b

# Get the nodes of the graph
ListNodes onlyTwo