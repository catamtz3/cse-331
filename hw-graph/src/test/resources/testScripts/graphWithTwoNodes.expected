# Create a graph
created graph onlyTwo

#Add two nodes to the graph
added node a to onlyTwo
added node b to onlyTwo

# Get the nodes of the graph
onlyTwo contains: a b